ddtrace/internal/injection.py

```diff
@@ -30,7 +30,7 @@ def _inject_hook(code, hook, lineno, arg):
     identifier for the hook itself. This should be kept in case the hook needs
     to be removed.
     """
-    for i, instr in enumerate(code):
+    for _i, instr in enumerate(code):
```

Comment:
🤔  what is the problem here?

Reply:
Probably 
```
B007: Loop control variable not used within the loop body. If this is intended, start the name with an underscore.
```
?

Reply:
ah right! it is used afterwards, but I suppose bugbear can't tell 🙁

Reply:
Another instance of `B007: Loop control variable not used within the loop body. If this is intended, start the name with an underscore.`